application/fpga/python/DobotDriver.py

```diff
@@ -495,6 +497,29 @@ def LaserOn(self, on):
 			result = self._write1(CMD_LASER_ON, 0)
 		self._lock.release()
 		return result
+	
+	def PumpOn(self, on):
+		'''
+		'''
```

Comment:
That's where my understanding of how they actually did it falls short, I suppose the motor with all the tubes attached is the actual device "pumping" air but what is the smaller one ? what is its purpose ? is it opening/closing the exit of the tube ?

PS: These might be stupid questions but I have never played with air before ^^